packages/table/src/tableBody2.tsx

```diff
@@ -19,20 +19,54 @@ import * as React from "react";
 
 import { AbstractComponent, ContextMenu, type ContextMenuContentProps, Utils as CoreUtils } from "@blueprintjs/core";
 
-import type { CellCoordinates } from "./common/cellTypes";
+import type { CellCoordinates, FocusedRegion, FocusMode } from "./common/cellTypes";
 import * as Classes from "./common/classes";
 import { toFocusedRegion } from "./common/internal/focusedCellUtils";
 import { RenderMode } from "./common/renderMode";
 import type { CoordinateData } from "./interactions/dragTypes";
-import { MenuContextImpl } from "./interactions/menus";
-import { DragSelectable } from "./interactions/selectable";
+import { type ContextMenuRenderer, MenuContextImpl } from "./interactions/menus";
+import { DragSelectable, type SelectableProps } from "./interactions/selectable";
+import type { Locator } from "./locator";
 import { type Region, Regions } from "./regions";
-import type { TableBodyProps } from "./tableBody";
-import { TableBodyCells } from "./tableBodyCells";
+import { TableBodyCells, type TableBodyCellsProps } from "./tableBodyCells";
 
 const DEEP_COMPARE_KEYS: Array<keyof TableBodyProps> = ["selectedRegions"];
 
-export class TableBody2 extends AbstractComponent<TableBodyProps> {
+export interface TableBodyProps extends SelectableProps, TableBodyCellsProps {
```

Comment:
Moved this type here from `tableBody.tsx` since that results in the smallest diff. In a follow up PR I will rename `tableBody2.tsx`. Didn't do it in this PR as that would result in a bad diff/blame.